client.cpp

```diff
@@ -490,3 +491,12 @@ PHP_METHOD(VoltClient, drain)
         RETURN_FALSE;
     }
 }
+
+PHP_METHOD(VoltClient, close)
+{
+    zval *zobj = getThis();
+    voltclient_object *obj = (voltclient_object *)zend_object_store_get_object(zobj TSRMLS_CC);
+
+    voltdb::ConnectionPool::pool()->returnClient(*(obj->client));
```

Comment:
You also want to catch the exceptions that can be thrown from the C++ library to make sure you translate those into the corresponding PHP exceptions.